h/templates/activity/search.html.jinja2

```diff
@@ -33,26 +33,20 @@
 {# Card displaying statistics about a bucket of annotations. #}
 {% macro search_bucket_stats(bucket) %}
 <div class="search-bucket-stats">
-  {% if bucket.uri %}
+  {% if bucket.uri and bucket.incontext_link(request) %}
     <div class="search-bucket-stats__key">
-        URL
-    </div>
-    <div class="search-bucket-stats__val search-bucket-stats__url">
-        <div><a class="search-bucket-stats__link"
-                href="{{ bucket.uri }}">{{ pretty_link(bucket.uri) }}</a></div>
-        {% if bucket.incontext_link(request) %}
-        <div>
-          <a class="search-bucket-stats__incontext-link"
-             href="{{ bucket.incontext_link(request) }}"
-             target="_blank">
-             Visit annotations in context {{ svg_icon('up-right-arrow', css_class='search-bucket-stats__incontext-icon') }}
-           </a>
-        </div>
-        {% endif %}
+      {{ svg_icon('up-right-arrow', css_class='search-bucket-stats__icon') }}
+      <a class="search-bucket-stats__incontext-link"
+         href="{{ bucket.incontext_link(request) }}"
+         target="_blank">
+         Visit annotations in context
+       </a>
     </div>
+    <div class="search-bucket-stats__val"></div>
```

Comment:
So the "Visit annotations in context" search bucket stats key doesn't actually have a value to go with it like the others do, but it requires a val div anyway for layout purposes?

I guess you could also wrap each key / value pair in a containing `<section>` or something and put the bottom margin on that instead of on the value div, but this may be simpler.